File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/AllyOceanDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.model.DragonType;

public class AllyOceanDragonKillAnimation extends StaticBlinkingAnimation {
    public AllyOceanDragonKillAnimation() {
        super(8, DragonType.OCEAN.getColor());
    }
}

File: src/hud/animations/AllyOceanDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {AllyDragonKillBlinkingAnimation} from './AllyDragonKillBlinkingAnimation.js';

export class AllyOceanDragonKillAnimation extends AllyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.OCEAN);
    }
}
